features/offer/offer-template-creation.feature

```diff
@@ -6,36 +6,36 @@ Feature: Create an offer template
 
   Scenario Outline: offer template is created
     Given the server is up
-    When an offer template is created in the "offer-template-game" game with:
```

Comment:
You can keep the table and in the godogs definitions you'll need something like this:

```
func anOfferTemplateIsCreatedInTheGameWith(gameID string, otArgs *gherkin.DataTable) error {
	payload := map[string]interface{}{
		"gameId":    gameID,
		"name":      otArgs.Rows[1].Cells[0].Value,
		"productId": otArgs.Rows[1].Cells[1].Value,
		"contents":  dat.JSON([]byte(otArgs.Rows[1].Cells[2].Value)),
		"metadata":  dat.JSON([]byte(otArgs.Rows[1].Cells[3].Value)),
		"period":    dat.JSON([]byte(otArgs.Rows[1].Cells[4].Value)),
		"frequency": dat.JSON([]byte(otArgs.Rows[1].Cells[5].Value)),
		"trigger":   dat.JSON([]byte(otArgs.Rows[1].Cells[6].Value)),
		"placement": otArgs.Rows[1].Cells[7].Value,
	}
	var err error
	lastStatus, lastBody, err = performRequest(app, "POST", "/offer-templates", payload)

	return err
}
```